modules/cnv-web-console-overview-dashboard.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+// * assembly/architecture
+
+[id="web-console-overview-dashboard_{context}"]
+= Overview Dashboard
+
+* *Details* card provides an brief overview of informational cluster details, including:
+** Cluster name
+** Provider
+** Version number of {CNVProductName} Container Platform.
```

Comment:
I think you mean to say OpenShift Container Platform here, right? I would just change this line to:
`** {product-title} version`

Reply:
Since these are GUI titles, I would use what is displayed and markup appropriately. I believe they should be: *Cluster ID*, *Provider*, and *OpenShift version* but confirm with ENG/QE.

Reply:
Made all changes. @rawagner can you confirm these labels are as @aburdenthehand states?

Reply:
yes, @aburdenthehand is correct

Reply:
Done, thanks guys.